res/values-es/strings.xml

```diff
@@ -5,27 +5,27 @@
     <string name="Apply">Aplicar</string>
     <string name="BangButtonExplanation">Buscar en otros lugares es fácil con !comandos</string>
     <string name="Cancel">Cancelar</string>
-    <string name="ChangingFontSize">Cambiando Tamaño de Letra&#8230;</string>
+    <string name="ChangingFontSize">Cambiando tamaño de letra&#8230;</string>
     <string name="Confirm">Confirmar</string>
-    <string name="ConfirmClearCacheAndCookies">El caché del navegador va a ser eliminado. ¿Estás seguro?</string>
-    <string name="ConfirmClearCookies">Todas las cookies del navegador serán eliminadas. ¿Estás seguro?</string>
-    <string name="ConfirmClearHistory"> Todo el historial de búsqueda será eliminado. ¿Estás seguro? </string>
-    <string name="ConfirmExternalBrowser">Por favor Confirma</string>
-    <string name="Continue">Continua</string>
+    <string name="ConfirmClearCacheAndCookies">Se eliminará el caché del navegador. ¿Estás seguro?</string>
+    <string name="ConfirmClearCookies">Se eliminarán todas las cookies del navegador. ¿Estás seguro?</string>
+    <string name="ConfirmClearHistory">Se eliminará todo el historial de búsqueda. ¿Estás seguro?</string>
+    <string name="ConfirmExternalBrowser">Por favor, confirma</string>
+    <string name="Continue">Continuar</string>
```

Comment:
`Por favor, confirma` is imperative form in spanish for `confirm`
`Continuar` is infinitive form in spanish for `continue`

An application using imperative form in spanish sounds kinda harsh. I'd change it to infinitive form of `confirm`:

`Confirmar`

That's just an option, but definitely imperative sounds weird.